<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="2" passed="0">
  <reporter-output>
    <line>
      <![CDATA[<a href='D:\Selenium and Java Labs\ZytoTestAutomation\target\surefire-reports\failure_screenshots\Click_On_Create_button_04_08_2020_01_28_05.png'> <img src='D:\Selenium and Java Labs\ZytoTestAutomation\target\surefire-reports\failure_screenshots\Click_On_Create_button_04_08_2020_01_28_05.png' height='100' width='100'/> </a>]]>
    </line>
    <line>
      <![CDATA[<a href='D:\Selenium and Java Labs\ZytoTestAutomation\target\surefire-reports\failure_screenshots\Click_On_Create_button_04_08_2020_01_29_15.png'> <img src='D:\Selenium and Java Labs\ZytoTestAutomation\target\surefire-reports\failure_screenshots\Click_On_Create_button_04_08_2020_01_29_15.png' height='100' width='100'/> </a>]]>
    </line>
  </reporter-output>
  <suite name="ZYTO AUTOMATION PARALLEL TEST EXECUTION SUITE" duration-ms="138890" started-at="2020-08-03T19:56:58Z" finished-at="2020-08-03T19:59:17Z">
    <groups>
    </groups>
    <test name="Zyto Automation  Parallel Test Execution -- chrome" duration-ms="68696" started-at="2020-08-03T19:56:58Z" finished-at="2020-08-03T19:58:07Z">
      <class name="com.zyto.test.base.ZytoBaseTest">
        <test-method status="FAIL" signature="setUp(java.lang.String)[pri:0, instance:com.zyto.test.base.ZytoBaseTest@2a4fb17b]" name="setUp" is-config="true" duration-ms="12470" started-at="2020-08-03T19:56:58Z" finished-at="2020-08-03T19:57:10Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.IllegalMonitorStateException">
            <full-stacktrace>
              <![CDATA[java.lang.IllegalMonitorStateException
at java.lang.Object.wait(Native Method)
at com.zyto.test.base.ZytoBaseTest.setUp(ZytoBaseTest.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalMonitorStateException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
      </class> <!-- com.zyto.test.base.ZytoBaseTest -->
      <class name="com.zyto.test.TestCases.Create_New_Account_Test">
        <test-method status="PASS" signature="SetUp()[pri:0, instance:com.zyto.test.TestCases.Create_New_Account_Test@5c6648b0]" name="SetUp" is-config="true" duration-ms="24827" started-at="2020-08-03T19:57:10Z" finished-at="2020-08-03T19:57:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method status="FAIL" signature="Click_On_Create_button()[pri:0, instance:com.zyto.test.TestCases.Create_New_Account_Test@5c6648b0]" name="Click_On_Create_button" duration-ms="29421" started-at="2020-08-03T19:57:35Z" finished-at="2020-08-03T19:58:05Z">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"xpath","selector":"//input[@value='es']"}
  (Session info: chrome=84.0.4147.105)
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'LAPTOP-QBC6F12R', ip: '192.168.1.104', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_172'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 84.0.4147.105, chrome: {chromedriverVersion: 84.0.4147.30 (48b3e868b4cc0..., userDataDir: C:\Users\Naresh\AppData\Loc...}, goog:chromeOptions: {debuggerAddress: localhost:54042}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: d8102dbbc0afca1cdd4b71af2a8793b1
*** Element info: {Using=xpath, value=//input[@value='es']}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//input[@value='es']"}
  (Session info: chrome=84.0.4147.105)
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'LAPTOP-QBC6F12R', ip: '192.168.1.104', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_172'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 84.0.4147.105, chrome: {chromedriverVersion: 84.0.4147.30 (48b3e868b4cc0..., userDataDir: C:\Users\Naresh\AppData\Loc...}, goog:chromeOptions: {debuggerAddress: localhost:54042}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: d8102dbbc0afca1cdd4b71af2a8793b1
*** Element info: {Using=xpath, value=//input[@value='es']}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:164)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:371)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:473)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy22.isSelected(Unknown Source)
at com.zyto.test.pages.Signup_Options_page.Change_Language(Signup_Options_page.java:97)
at com.zyto.test.TestCases.Create_New_Account_Test.Click_On_Create_button(Create_New_Account_Test.java:92)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
            <line>
              <![CDATA[<a href='D:\Selenium and Java Labs\ZytoTestAutomation\target\surefire-reports\failure_screenshots\Click_On_Create_button_04_08_2020_01_28_05.png'> <img src='D:\Selenium and Java Labs\ZytoTestAutomation\target\surefire-reports\failure_screenshots\Click_On_Create_button_04_08_2020_01_28_05.png' height='100' width='100'/> </a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- Click_On_Create_button -->
        <test-method status="PASS" signature="EndTest()[pri:0, instance:com.zyto.test.TestCases.Create_New_Account_Test@5c6648b0]" name="EndTest" is-config="true" duration-ms="659" started-at="2020-08-03T19:58:06Z" finished-at="2020-08-03T19:58:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- EndTest -->
      </class> <!-- com.zyto.test.TestCases.Create_New_Account_Test -->
    </test> <!-- Zyto Automation  Parallel Test Execution - chrome -->
    <test name="Zyto Automation  Parallel Test Execution -- Firefox" duration-ms="69891" started-at="2020-08-03T19:58:07Z" finished-at="2020-08-03T19:59:17Z">
      <class name="com.zyto.test.base.ZytoBaseTest">
        <test-method status="FAIL" signature="setUp(java.lang.String)[pri:0, instance:com.zyto.test.base.ZytoBaseTest@618b19ad]" name="setUp" is-config="true" duration-ms="14513" started-at="2020-08-03T19:58:07Z" finished-at="2020-08-03T19:58:21Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefox]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.IllegalMonitorStateException">
            <full-stacktrace>
              <![CDATA[java.lang.IllegalMonitorStateException
at java.lang.Object.wait(Native Method)
at com.zyto.test.base.ZytoBaseTest.setUp(ZytoBaseTest.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.beforeRun(TestRunner.java:529)
at org.testng.TestRunner.run(TestRunner.java:497)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalMonitorStateException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
      </class> <!-- com.zyto.test.base.ZytoBaseTest -->
      <class name="com.zyto.test.TestCases.Create_New_Account_Test">
        <test-method status="PASS" signature="SetUp()[pri:0, instance:com.zyto.test.TestCases.Create_New_Account_Test@2d3379b4]" name="SetUp" is-config="true" duration-ms="23879" started-at="2020-08-03T19:58:21Z" finished-at="2020-08-03T19:58:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SetUp -->
        <test-method status="FAIL" signature="Click_On_Create_button()[pri:0, instance:com.zyto.test.TestCases.Create_New_Account_Test@2d3379b4]" name="Click_On_Create_button" duration-ms="29506" started-at="2020-08-03T19:58:45Z" finished-at="2020-08-03T19:59:15Z">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"xpath","selector":"//input[@value='es']"}
  (Session info: chrome=84.0.4147.105)
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'LAPTOP-QBC6F12R', ip: '192.168.1.104', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_172'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 84.0.4147.105, chrome: {chromedriverVersion: 84.0.4147.30 (48b3e868b4cc0..., userDataDir: C:\Users\Naresh\AppData\Loc...}, goog:chromeOptions: {debuggerAddress: localhost:54216}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: 5a25778bbb7cca914c06656ffd0b9188
*** Element info: {Using=xpath, value=//input[@value='es']}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//input[@value='es']"}
  (Session info: chrome=84.0.4147.105)
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.14.0', revision: 'aacccce0', time: '2018-08-02T20:19:58.91Z'
System info: host: 'LAPTOP-QBC6F12R', ip: '192.168.1.104', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_172'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 84.0.4147.105, chrome: {chromedriverVersion: 84.0.4147.30 (48b3e868b4cc0..., userDataDir: C:\Users\Naresh\AppData\Loc...}, goog:chromeOptions: {debuggerAddress: localhost:54216}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
Session ID: 5a25778bbb7cca914c06656ffd0b9188
*** Element info: {Using=xpath, value=//input[@value='es']}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:164)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:371)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:473)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy22.isSelected(Unknown Source)
at com.zyto.test.pages.Signup_Options_page.Change_Language(Signup_Options_page.java:97)
at com.zyto.test.TestCases.Create_New_Account_Test.Click_On_Create_button(Create_New_Account_Test.java:92)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
            <line>
              <![CDATA[<a href='D:\Selenium and Java Labs\ZytoTestAutomation\target\surefire-reports\failure_screenshots\Click_On_Create_button_04_08_2020_01_29_15.png'> <img src='D:\Selenium and Java Labs\ZytoTestAutomation\target\surefire-reports\failure_screenshots\Click_On_Create_button_04_08_2020_01_29_15.png' height='100' width='100'/> </a>]]>
            </line>
          </reporter-output>
        </test-method> <!-- Click_On_Create_button -->
        <test-method status="PASS" signature="EndTest()[pri:0, instance:com.zyto.test.TestCases.Create_New_Account_Test@2d3379b4]" name="EndTest" is-config="true" duration-ms="776" started-at="2020-08-03T19:59:16Z" finished-at="2020-08-03T19:59:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- EndTest -->
      </class> <!-- com.zyto.test.TestCases.Create_New_Account_Test -->
    </test> <!-- Zyto Automation  Parallel Test Execution - Firefox -->
  </suite> <!-- ZYTO AUTOMATION PARALLEL TEST EXECUTION SUITE -->
</testng-results>
